src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -813,6 +816,29 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
     );
   };
 
+  isLoginForDownloadEnabled = () => {
+    if (this.props.buttonConfig.loginForDownload) {
+      return true;
+    }
+    return false;
```

Comment:
minor: effectively, you can turn this 5 line code block into an one-liner

this is the same as `isLoginForDownloadEnabled = () => !!this.props.buttonConfig.loginForDownload`

and you can even conbine this function with the `isDownloadButton` function down below and give it a more meaningful name. Since both of them are just returning a boolean value